### DISCUSSION TITLE
i just wanna filter my oders per customer

### DISCUSSION THREAD

**Discussion by IPolonio** [serenity-is/Serenity#7113] [Help]
### Before submitting the bug report, please read and check the following items

- [X] This isn't a feature request or a discussion topic (Please use discussions for that)
- [X] I have searched open and closed issues for duplicates
- [X] I have searched the documentation on the Serenity Platform website https://serenity.is/docs/
- [X] I have read the frequently asked questions https://serenity.is/docs/startsharp/faq
- [X] I have searched my problem in the discussions and the old issues https://github.com/serenity-is/Serenity/discussions

### What happened?

namespace MovieTutorial.CustomerPortal {
    @Serenity.Decorators.registerClass()
    export class OrderDetailEditDialog extends Common.GridEditorDialog<OrderDetailsRow> {
        protected getFormKey() { return OrderDetailsForm.formKey; }
        protected getIdProperty() { return OrderDetailsRow.idProperty; }
        protected getLocalTextPrefix() { return OrderDetailsRow.localTextPrefix; }
        protected form: OrderDetailsForm;
        private orderd: OrderDetailEdit
        

        constructor() {
            super();
            this.form = new OrderDetailsForm(this.idPrefix);
        }

      

        protected afterLoadEntity() {
            super.afterLoadEntity();
            this.orderd.personID = this.entityId;
            
        }
    }
}    

namespace MovieTutorial.CustomerPortal {
    @Serenity.Decorators.registerEditor()
    export class OrderDetailEdit extends Serenity.EntityGrid<OrderDetailsRow, any> {
        protected getColumnsKey() { return "CustomerPortal.OrderDetails"; }
        protected getDialogType() { return OrderDetailsDialog; }
        protected getIdProperty() { return OrderDetailsRow.idProperty; }
        protected getLocalTextPrefix() { return OrderDetailsRow.localTextPrefix; }
        protected getService() { return OrderDetailsService.baseUrl; }
       
        private _personID: number;

        

        get personID() {
            return this._personID;
            console.log(this.personID);
        }

        set personID(value: number) {
            if (this._personID != value) {
                this._personID = value;
                this.setEquality(OrderDetailsRow.Fields.OrderId, value);
                this.refresh();

            }
            console.log(this.personID);
        }
        
        constructor(container: JQuery) {
            super(container);
        }

        protected getButtons() {
            return null;
        }

        protected getInitialTitle() {
            return null;
        }

        protected usePager() {
            return false;
        }

        protected getGridCanLoad() {
            return this.personID != null;
            
        }
    }
}

### What did you expect to happen?

its expected to separate the orders by customers

### How to reproduce?

just run my code

### What Serenity Nuget Versions are you seeing the problem on? (separated by comma)

Serenity Obsolete 2.0

**Comment by LSNicholls** [comment]
So, your personID represents your customer here?  If so, what do you expect this line to be doing:

` this.setEquality(OrderDetailsRow.Fields.OrderId, value);`

... it doesn't reference the personID at all.  

Pardon me if this is a stupid question, I may not understand what you are trying to do.
